src/components/results/detail/AudioDetail.vue

```diff
@@ -29,102 +29,37 @@
               >
                 <!-- Content -->
                 <div>
-                  <MediaHeader
-                    :file="$props.file"
-                  />
```

Comment:
Would it be a big change to integrate what's special about `AudioDetail` so we consistently use `MediaHeader` in most places?